src/taipy/gui_core/_context.py

```diff
@@ -69,9 +81,22 @@ def __init__(self, gui: Gui) -> None:
         self.data_nodes_by_owner: t.Optional[t.Dict[t.Optional[str], DataNode]] = None
         self.scenario_configs: t.Optional[t.List[t.Tuple[str, str]]] = None
         self.jobs_list: t.Optional[t.List[Job]] = None
+        self.client_jobs_by_submission: t.Dict[
+            str,
+            t.Tuple[
```

Comment:
Maybe use a namedtuple here to better follow what's going on?

Reply:
really ?

Reply:
changed to a class